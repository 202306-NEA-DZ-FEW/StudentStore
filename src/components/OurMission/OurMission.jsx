import Image from "next/image";
export default function OurMission({ t }) {
    return (
        <div>
            <div className='flex flex-col md:flex-row-reverse'>
                <div className='w-full md:w-1/2 md:flex md:items-center'>
                    <Image
                        src='/images/aboutus.svg'
                        alt='Our Mission Photo'
                        loading='lazy'
                        width={1200}
                        height={1200}
                        layout='responsive'
                    />
                </div>
                <div className='flex flex-col justify-center items-center gap-3 w-full md:w-1/2 p-4'>
                    <h1 className='text-4xl text-[#32314D] font-bold'>
                        {t("Our Mission")}
                    </h1>
                    <p className='text-center text-[#32314D] md:text-md lg:text-lg leading-8 md:leading-5 '>
                        {t("Student Store")}
                    </p>
                </div>
            </div>
        </div>
    );
}
