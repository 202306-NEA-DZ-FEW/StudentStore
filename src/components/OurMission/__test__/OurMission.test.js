import renderer from "react-test-renderer";
import OurMission from "../OurMission";

it("renders correctly", () => {
    const tree = renderer.create(<OurMission />).toJSON();
    expect(tree).toMatchSnapshot();
});
