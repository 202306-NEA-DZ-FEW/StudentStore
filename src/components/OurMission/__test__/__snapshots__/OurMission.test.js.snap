// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="flex flex-col md:flex-row-reverse"
  >
    <div
      className="w-full md:w-1/2 md:flex md:items-center"
    >
      <img
        alt="Our Mission Photo"
        data-nimg="1"
        decoding="async"
        height={1200}
        loading="lazy"
        onError={[Function]}
        onLoad={[Function]}
        src="/images/aboutus.svg"
        style={
          Object {
            "color": "transparent",
            "height": "auto",
            "width": "100%",
          }
        }
        width={1200}
      />
    </div>
    <div
      className="flex flex-col justify-center items-center gap-3 w-full md:w-1/2 p-4"
    >
      <h1
        className="text-4xl text-[#32314D] font-bold"
      >
        Our Mission
      </h1>
      <p
        className="text-center text-[#32314D] md:text-md lg:text-lg leading-8 md:leading-5 "
      >
        Student Store
      </p>
    </div>
  </div>
</div>
`;
